src/app/plan/duck/operations.ts

```diff
@@ -64,13 +65,23 @@ const fetchPlan = () => {
   };
 };
 
-const fetchNamespacesForCluster = (clusterName) => {
+const fetchNamespacesForCluster = clusterName => {
   return (dispatch, getState) => {
-    const client: IClusterClient = ClientFactory.forCluster(clusterName, getState());
-    const nsResource = new CoreClusterResource(CoreClusterResourceKind.Namespace);
-    client.list(nsResource).then(res => {
-      dispatch(sourceClusterNamespacesFetchSuccess(res.data.items));
-    }).catch(err => AlertCreators.alertError('Failed to load namespaces for cluster'));
+    const client: IClusterClient = ClientFactory.forCluster(
+      clusterName,
+      getState(),
+    );
+    const nsResource = new CoreClusterResource(
+      CoreClusterResourceKind.Namespace,
+    );
+    client
```

Comment:
Please, use async/await

Reply:
Is there any reason, not use here async/await?

Reply:
this is not related to this PR. Please open an issue. The only changes made here were from a linting tool.

Reply:
Ok